apps/frontend-pwa/src/components/liveSession/QuestionArea.tsx

```diff
@@ -0,0 +1,311 @@
+import { push } from '@socialgouv/matomo-next'
+import { H1 } from '@uzh-bf/design-system'
+import dayjs from 'dayjs'
+import localForage from 'localforage'
+import _without from 'lodash/without'
+import React, { useEffect, useState } from 'react'
+import v8n from 'v8n'
```

Comment:
I think we use several different validation libraries across the code base. should try to find a common library that works everywhere. validator, v8n, yup, etc.

Reply:
replaced v8n with yup and removed corresponding dependency